packages/vaadin-lumo-styles/src/components/tab.css

```diff
@@ -0,0 +1,251 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+:host {
+  box-sizing: border-box;
+  padding: 0.5rem 0.75rem;
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--lumo-line-height-xs);
+  font-weight: 500;
+  opacity: 1;
+  color: var(--lumo-secondary-text-color);
+  transition:
+    0.15s color,
+    0.2s transform;
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  position: relative;
+  cursor: var(--lumo-clickable-cursor);
+  transform-origin: 50% 100%;
+  outline: none;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  overflow: hidden;
+  min-width: var(--lumo-size-m);
+  -webkit-user-select: none;
+  user-select: none;
+  --_focus-ring-color: var(--vaadin-focus-ring-color, var(--lumo-primary-color-50pct));
+  --_focus-ring-width: var(--vaadin-focus-ring-width, 2px);
+  --_selection-color: var(--vaadin-selection-color, var(--lumo-primary-color));
+  --_selection-color-text: var(--vaadin-selection-color-text, var(--lumo-primary-text-color));
+}
+
+:host([hidden]) {
+  display: none !important;
+}
+
+@media (forced-colors: active) {
+  :host([focused]) {
+    outline: 1px solid;
+    outline-offset: -1px;
+  }
+
+  :host([selected]) {
+    border-bottom: 2px solid;
+  }
+}
+
+:host(:not([orientation='vertical'])) {
+  text-align: center;
+}
+
+:host([orientation='vertical']) {
+  transform-origin: 0% 50%;
+  padding: 0.25rem 1rem;
+  min-height: var(--lumo-size-m);
+  min-width: 0;
+}
+
+@media (forced-colors: active) {
+  :host([focused]) {
+    outline: 1px solid;
+    outline-offset: -1px;
+  }
```

Comment:
This code block seems duplicated, see line 41 above. Also `@media (forced-colors: active)` could be combined.